src/parkapi_sources/converters/herrenberg_bike/models.py

```diff
@@ -112,7 +111,8 @@ class HerrenbergBikePropertiesInput(ValidataclassMixin):
     operator_name: OptionalUnset[str] = NoneToUnsetValue(StringValidator(min_length=0, max_length=256)), DefaultUnset
     capacity: int = IntegerValidator(min_value=0)
     capacity_charging: OptionalUnset[int] = NoneToUnsetValue(IntegerValidator(min_value=0)), DefaultUnset
-    capacity_cargobike: OptionalUnset[int] = NoneToUnsetValue(IntegerValidator(min_value=0)), DefaultUnset
+    # capacity_cargobike is unsupported yet
```

Comment:
Question: is it known when it will be supported? I am wondering if it would help to write the comment as a TODO + number of the issue where it will be fixed, so it's easier to find it later.

Reply:
Did not do a TODO, as it is no TODO atm, but linked the ticket: https://github.com/ParkenDD/parkapi-sources-v3/issues/174